macos/FluentUI/Button/Button.swift

```diff
@@ -264,6 +300,17 @@ open class Button: NSButton {
 		} else {
 			contentTintColor = contentTintColorRest
 		}
+		updateTrailingImageContentTintColor()
+	}
+
+	private func updateTrailingImageContentTintColor() {
+		if !isEnabled {
```

Comment:
can enclose this in `if let trailingImageView`? Then we don't need any optional unwraps in each of the cases and we save some trivial work if we don't have a trailing image